x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -38,7 +38,7 @@ testClusters.all {
   numberOfNodes = 3
   testDistribution = 'DEFAULT'
 
-  if (BuildParams.isSnapshotBuild() == false) {
```

Comment:
While a little more verbose, I think the current implementation is a bit more readable. However, I'll defer to our developers.